mosaic-site/src/components/app-header/AppHeader.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC, useEffect, useLayoutEffect, useState } from "react";
+import classnames from "classnames";
```

Comment:
I might be missing something but is classnames a transitive dependency? It's not defined in our package.json and we are using clsx in the components.

Reply:
we could move to clsx... be good to converge on common dependencies
@DavieReid  what's your thoughts ?

Reply:
I'll replace this with clsx as per our chat in symphony